### PR TITLE
Create walkthrough example with data

### PR DESCRIPTION
**Pull request by marko-polo-cheno**
This PR provides a quick guide with all needed data to get a useful leaderboard on AutoArena. The notebook explains the core principles of why AutoArena uses an Elo leaderboard, and shows how it works.

### CODE REVIEW

File: examples/assets/autojudge.png
Comment (gordonhart) [inline examples/assets/autojudge.png:1]: Very minor, but considering that these are checked into the source tree, smaller is better — would you mind transcoding to JPEG?

File: examples/assets/elo_scores.png
Comment (gordonhart) [inline examples/assets/elo_scores.png:1]: Based on the number of votes versus the number of responses, it looks like there are many unmatched prompts between these models. Can we pare down the CSVs to only include matched prompts?

**Comment by gordonhart** [comment]
When I run this myself, I get `llama-2-70b` ranked above `gpt-4-0613` — have you seen this? It's an unexpected result, that might be due to the particular prompts+responses included in the CSVs.

Screenshot 2024-10-04 at 11 22 38 AM

Edit: looks like the culprit is a heavily skewed H2H distribution. If `gpt-4-0613` has the lion's share of its H2Hs against the champion, it'll look worse than it really is on the main leaderboard:

Screenshot 2024-10-04 at 11 23 42 AM

File: examples/assets/elo_scores.png
Comment (marko-polo-cheno) [reply examples/assets/elo_scores.png:1]: I can pare down the CSVs, but unfortunately there are very few prompts shared amongst all models.

**Comment by marko-polo-cheno** [comment]
> When I run this myself, I get `llama-2-70b` ranked above `gpt-4-0613` — have you seen this? It's an unexpected result, that might be due to the particular prompts+responses included in the CSVs.

This is unexpected. Opponents shouldn't be allowed to "farm" Elo off of each other (gpt-4-1106-preview is _untouchable_).

**Review by gordonhart — APPROVED** [review]

**Pull request merged**